GVFS/GVFS.Tests/Utilities.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+namespace GVFS.Tests
+{
+    public class Utilities
```

Comment:
'TestFixtureUtilities'? Do we expect to have many helpers like this, or should we use a generic Utilities name?

Also, I'm open to moving this out of GVFS.Tests but it seemed like a nice fit with NUnitRunner.cs living there.